js/views/catView.js

```diff
@@ -0,0 +1,64 @@
+const View = {
+    init: function() {
+        // aca va todo lo que agarra las cosas del dom
+        this.catImg = document.getElementById('cat-img');
```

Comment:
Encapsulate this logic: `agarraTodoloDelDomById('id1', 'id2', 'id3') {}`
Inicializa las variables catImg, catName y catCount fuera de esta función (a nivel de `View`) y asigna los valores iniciales dentro de 'agarra...'